docs/end-to-end-local-email-dev.md

```diff
@@ -16,12 +17,13 @@ sequenceDiagram
 
 ## Requirements
 
-* Your own domain and the ability to publish MX and CNAME records to it
-* AWS account
-* (Suggested) [ngrok.io][ngrok] account
-* Enable Mozilla Accounts authentication (see README)
+- Your own domain and the ability to publish MX and CNAME records to it
+- AWS account
+- (Suggested) [ngrok.io][ngrok] account
+- Enable Mozilla Accounts authentication (see README)
```

Comment:
suggestion (blocking): Since markdown supports asterisks, pluses, and hyphens interchangeably for bulleted lists, let's leave the characters un-changed as much as we can to reduce code churn in our PRs.